features/navigator.yml

```diff
@@ -0,0 +1,6 @@
+name: Navigator
+description: "The `window.navigator` API represents the user agent and its state. It's also the generic global object under which many other APIs are located, such as `clipboard` or `wakeLock`."
+spec: https://html.spec.whatwg.org/multipage/system-state.html#the-navigator-object
+compat_features:
+  - api.Navigator
+  - api.Window.navigator
```

Comment:
I went with the latter approach, since hiding away such a prominent piece of the web in an unrelated feature doesn't seem right. Better to have it visible as a signal of future intent (even if explicitly saying it does nothing on its own).